pkg/scheduler/framework/interface_test.go

```diff
@@ -27,7 +27,7 @@ import (
 
 var errorStatus = NewStatus(Error, "internal error")
 
-func TestStatus(t *testing.T) {
+func TestIsSuccessStatus(t *testing.T) {
```

Comment:
add `expectedIsWait bool` in the struct

Reply:
thx.
this one seems simpler and easier to write tests for.